### DISCUSSION TITLE
ubuntu下如何只使用这个的配置文件

### DISCUSSION THREAD

**Discussion by wangrfx** [hooke007/mpv_PlayKit#91] [mpv 通用]
没研究过mpv的详细配置，有人知道如何仅使用配置文件吗

**Comment by hooke007** [comment]
鉴于部分脚本是windows only的，只讨论文件目录结构和主设置文件，你至少需要做的：
- 参考 https://mpv.io/manual/master/#files 迁移 `portable config` 中的文件到目标路径
- 修改 mpv.conf 中windows相关的参数

**Comment by dyphire** [comment]
linux下使用要改动的地方挺多的，还是建议参考手册自己写配置

**Comment by nyfair** [comment]
相比windows only的部分，trunk build only的部分更多
这个配置比较偏向新引入的功能，ubuntu官方源里不仅版本停留在上上个稳定版0.33，而且编译的时候没有启用vapoursynth和shaderc这俩被大量使用到的依赖，很多地方都会报警告而不可用

**Reply by nyfair** [reply]
有需要可以尝试自己编译或者这个ppa https://launchpad.net/~savoury1/+archive/ubuntu/mpv

**Comment by txyyh** [comment]
Arch 用户报道

仅仅修改gpu-context,ao,hwdec等接口相关部分，即可让配置运行起来

不过要想完美运行，你得安装大量所需的依赖,并尽可能使用git版本的mpv
